SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -173,7 +173,7 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	int32 GetConsiderListSize() const { return ConsiderListSize; }
 #endif
 
-	void DelayedSendDeleteEntityRequest(Worker_EntityId EntityId, float Delay);
+	void DelayedSendDeleteEntityRequest(Worker_EntityId EntityId, float Delay, bool bIsNetStartupActor);
```

Comment:
Could we similarly name this DelayedRetireEntity? The name here is misleading.

Reply:
oh rename the function to that

Reply:
👌